### DISCUSSION TITLE
Show errors and warnings pop-up menu broken

### DISCUSSION THREAD

**Discussion by wwxodus** [VonHeikemen/lsp-zero.nvim#140] [Q&A] [answered]
I dont know what's happen, but `gl` command doesn't work for me when I press it on error or warning line((((
I tried to remap it, but didn't find any information about it

**Comment by VonHeikemen** [comment, marked as answer]
Is the mapping active? 

If you execute the command `:map gl` what do you get?

**Reply by wwxodus** [reply]
I find good solution, this solution was being in lsp-zero documentation))))
